change/@azure-msal-browser-9df1850b-ffe9-4027-969f-dad511cc00f6.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "none",
```

Comment:
```suggestion
  "type": "patch",
```

Reply:
Well, is it really a patch? We are not fixing a bug, it is purely code refactor.

Reply:
Any change to published code is not a `none` change

Reply:
Done.